src/DataProtection/README.md

```diff
@@ -92,6 +92,14 @@ directive:
       verb: Get
       subject: RecoveryPointList.*
     hide: true
+  - where:
```

Comment:
please add branch tag to lock the code of https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2022-05-01/dataprotection.json in certain commit id.

Reply:
Done